tst/EnergyPlus/unit/WaterThermalTanks.unit.cc

```diff
@@ -1992,7 +1992,7 @@ TEST_F(EnergyPlusFixture, StratifiedTankCalc)
     for (int i = 0; i < Tank.Nodes; ++i) {
         NodeTemps[i] = Tank.Node[i].Temp;
     }
-
+    
```

Comment:
I'm not sure what editor you are using to write C++, but it shouldn't be leaving these trailing whitespaces in the files.  In most editors there is an option to clean any trailing whitespace when you save a file, I'd look for that option.  I'm not going to hold up this PR just for that though.